static-typing/experiment.py

```diff
@@ -0,0 +1,66 @@
+from typing import Any
```

Comment:
This is where the value is :-) Showing how `dataclass_transform` can be applied etc.